homeassistant/components/unifi/sensor.py

```diff
@@ -100,3 +101,25 @@ def state(self) -> int:
         if self._is_wired:
             return self.client.wired_tx_bytes / 1000000
         return self.client.tx_bytes / 1000000
+    
+    
+class UniFiUpTimeSensor(UniFiClient):
+    """UniFi uptime sensor base class."""
+
+    DOMAIN = DOMAIN
+    TYPE = UPTIME_SENSOR
+    
+    @property
+    def name(self) -> str:
+        """Return the name of the client."""
+        return f"{super().name} {self.TYPE.upper()}"
+
+    @property
+    def unit_of_measurement(self) -> str:
+        """Return the unit of measurement of this entity."""
+        return TIME_SECONDS
```

Comment:
This should be a timestamp sensor instead without unit.
https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes